design/authenticators/authn_oidc/oidc_configuration_enhancements.md

```diff
@@ -0,0 +1,228 @@
+# OIDC Authenticator Configuration Improvements
+
+## Issue Description
+There have been multiple instances where users of the OIDC authenticator have experienced limitations resulting 
+from our use of the third-party [OpenIDConnect gem](https://github.com/nov/openid_connect). The issues mainly stem 
+from the gem limiting changes to the HTTP config used for outgoing connections. So far the following problems have 
+been identified:
+- Unable to configure CA certs from the authenticator config in Conjur
+  - The workaround for custom CA certs is to add them to the Conjur container OpenSSL truststore (`/etc/ssl/certs`)
+- The underlying HTTP client does not honor the `HTTPS_PROXY` environment variable
```

Comment:
Are we supporting both `http_proxy` and `https_proxy`?  

Also, I believe these variables are typically lower case, can we check for uppercase or lowercase?

Reply:
That's a function of the underlying HTTP library. After making the gem update it will be Faraday with a Net::HTTP adapter. In my testing it honored both environment variables.